09_dt_rtc/rtc_ops.log

```diff
@@ -0,0 +1,33 @@
+/ # 
+/ # 
+/ # dmesg | grep 'rtc'
+[    2.079035] rtc-ds1307 2-0068: registered as rtc0
+[    2.161474] rtc-ds1307 2-0068: setting system clock to 2021-03-18 14:49:21 UTC (1616078961)
+/ # 
+/ # 
+/ # date -s "2021-03-18 14:50:00" 
+Thu Mar 18 14:50:00 UTC 2021
+/ # hwclock -w -f /dev/rtc0 
```

Comment:
Just note: `date -s` should set time in the default hw clock. You added rtc in the device tree and the module is loaded during boot time.